src/App.js

```diff
@@ -0,0 +1,90 @@
+// src/App.js
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import Home from './components/home';
+import Signup from './auth/signup';
+import Login from './auth/login';
+import { checkLoginStatus } from './redux/loginSlice';
+import Navbar from './components/navbar';
+import AddItem from './components/addItem';
+import ItemDetails from './components/itemDetails';
+import AddReservations from './components/addReservations';
+import DisplayReservations from './components/displayReservations';
+import DeleteItems from './components/deleteItems';
+import './stylesheets/App.css';
+
+function App() {
```

Comment:
- [ ] When I visit "reservations" or "delete reservations" the frontend goes into an infinite loop making about 50 requests to the server each second:

![image](https://github.com/geekelo/fenaj-schedula/assets/57421823/fad0188a-8b2b-46eb-b1df-d299923e900b)


The problem gets solved when you visit "home" or reload the page.

Kindly, fix this issue so the server gets requested once.

_tip:  This is normally because of a `useEffect ` hook that has met an edge case._